examples/simple/src/Simple.es6

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import Valya from '../../../build/index';
```

Comment:
I think about separate repo and with these thoughts I started this PR to discuss with you, guys.